README.md

```diff
@@ -1 +1,16 @@
-# Room_Occupancy_Project
\ No newline at end of file
+# Room Occupancy Detection
+## Overview
+- Selected Topic: Room occupancy detection for home or office using a variety of sensors such as temperature, motion, and humidity.  Datasets from multiple experiments have been captured and provide on Kaggle and the UCI machine learning datasets site.
+- Reasons for topic selection: The domain (people in a room) is something everyone can easily understand, the type of analysis needed (binary classification) is one of the skills we learned in this course, and availability of multiple, well formed datasets.  Potential use cases for occupancy detection include: 
```

Comment:
Reasons for topic selection: The domain (whether people are in a room) is an easily understood and widely useful question. The type of analysis needed (binary classification) is one of the skills we learned in this course. There are multiple, well formed datasets available to the public on this topic.

Reply:
Updated.